dataflux_pytorch/benchmark/checkpointing/multinode/train.py

```diff
@@ -95,42 +96,46 @@ def main(ckpt_dir_path: str, ckpt_restore_path: str = ""):
     if torch.distributed.get_rank() == 0:
         print(f"Saved checkpoint to {ckpt_dir_path} {num_save_calls} times.")
     avg_save_time = (end - start) / num_save_calls
-    num_load_calls = int(os.environ.get("NUM_LOAD_CALLS", 3))
-    load_checkpoint_times = []
-    for i in range(num_load_calls):
-        model = DemoTransformer(vocab_size=dataset.vocab_size,
-                                nlayers=int(os.environ.get("NUM_LAYERS", 10)))
-        new_ckpt_dir_path = os.path.join(ckpt_restore_path, f'ckpt_{i}.ckpt/')
-        strategy = get_strategy(args.strategy, os.getenv("PROJECT"), model,
-                                new_ckpt_dir_path)
-        trainer = Trainer(
-            enable_checkpointing=False,
-            default_root_dir=ckpt_dir_path,
-            plugins=[],
-            min_epochs=1,
-            max_epochs=1,
-            max_steps=1,
-            accelerator="gpu",
-            strategy=strategy,
-            devices=os.environ.get("NUM_DEVICES", 'auto'),
-            num_nodes=num_nodes,
-        )
-        trainer.fit(model, dataloader, ckpt_path=new_ckpt_dir_path)
-        start = time.time()
-        trainer.strategy.load_checkpoint(new_ckpt_dir_path)
-        end = time.time()
+    if not args.save_only:
+        num_load_calls = int(os.environ.get("NUM_LOAD_CALLS", 3))
```

Comment:
Could you not just set NUM_LOAD_CALLS to 0 to get effectively this same behavior?

Reply:
Done! Kept the `save_only` flag because it's needed to skip computing mean load times. `statistics.mean()` expects a non-empty array.

Reply:
Done.